props/blaster.h

```diff
@@ -108,7 +108,7 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
   }
 
   virtual bool CheckJam(int percent) {
-    int random = rand() % 100;
+    int random = rand() % 100 + 1;
```

Comment:
Good find, but it would be better to just change the next line to:

return random < percent;

Reply:
ah yes, that was the original thing I was going to do but I thought 100 percent passed in wouldn't function correctly but I think it will after looking again